client/js/components/input.js

```diff
@@ -6,30 +6,28 @@ import Spacer from '../components/spacer'
 const Input = ({ classes, input, label, type, placeholder }) => {
 	return (
 		<div className={classes.child}>
-			<label className={classes.label}>{label}</label>
-			<input className={classes.textInput} type={type} placeholder={placeholder} {...input} />
+			<input
+				className={classes.textInput}
+				type={type}
+				placeholder={placeholder}
+				{...input}
+			/>
 			<Spacer height="10px" />
 		</div>
 	)
 }
 
 const styles = {
 	child: {
-		'justify-content': 'center',
-		width: '80%',
-	},
-	label: {
-		padding: '20px 10px',
-		display: 'block',
-		float: 'left',
-		width: '20%',
-		'text-align': 'right',
+		display: 'flex',
+		'align-items': 'flex-start',
+		width: '100%',
 	},
 	textInput: {
-		width: '50%',
+		width: '100%',
 		padding: '12px 20px',
-		margin: '8px 0',
-		'font-size': '90%',
+		margin: '8px 30px 8px 0px',
+		'font-size': '100%',
```

Comment:
Let's set this to a default of `18px` since that's what the designs show for the majority of the input fields.